src/faq.mdx

```diff
@@ -5,14 +5,14 @@ USACO stands for the [USA Computing Olympiad](http://www.usaco.org/).
 ## Q: How Do USACO Contests Work?
 
 <Resources>
-  <Resource
-    source="USACO@USA Computing Olympiad"
-    title="Contests"
-    url="http://www.usaco.org/index.php?page=contests"
-    starred
-  >
-    more about contest format
-  </Resource>
+	<Resource
```

Comment:
did this file need to be modified? also i think 2 space indents are fine for source code

Reply:
idk it just happened when i merged